apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -414,6 +417,108 @@ static void tc_task_prctl(void)
 	TC_ASSERT_EQ_ERROR_CLEANUP("prctl", strncmp(getname, setname, CONFIG_TASK_NAME_SIZE), 0, get_errno(), prctl(PR_SET_NAME, oldname, 0, 0, 0));
 
 	prctl(PR_SET_NAME, oldname, 0, 0, 0);
+
+#ifdef CONFIG_MESSAGING_IPC
+	const char *portname = "testPort";
+	pid_t pid;
+	int recv_cnt = 0;
+	int *data_arr;
+	struct sched_param param;
+
+	/* saving receiver information */
+	pid = getpid();
+	ret_chk = sched_getparam(pid, &param);
+	TC_ASSERT_EQ("sched_getparam", ret_chk, OK);
+
+	ret_chk = prctl(PR_MSG_SAVE, portname, pid, param.sched_priority);
+	TC_ASSERT_EQ("prctl", ret_chk, OK);
+
+	/* Reading receiver information */
+
+	ret_chk = prctl(PR_MSG_READ, getname, data_arr, recv_cnt);
+	TC_ASSERT_NEQ("prctl", data_arr, NULL);
+	TC_ASSERT_NEQ("prctl", recv_cnt, ERROR);
+
+	/* Saving receiver information */
+
+	ret_chk = prctl(PR_MSG_SAVE, portname, pid, param.sched_priority);
+	TC_ASSERT_EQ("prctl", ret_chk, OK);
+
+	/* Removing receiver information */
+
+	ret_chk = prctl(PR_MSG_REMOVE, portname);
+	TC_ASSERT_EQ("prctl", ret_chk, OK);
+
+#endif
+#ifdef CONFIG_TASK_MONITOR
```

Comment:
This is duplicated with `tc_task_task_monitor_utils()`. So I think this is not needed.